src/workloads/transactions/templates/README.md

```diff
@@ -3,19 +3,26 @@ Long Lived Transactions
 
 Currently, genny does not support parameterization, but the LLT workloads are highly repetitive.
 
-For the moment, the __LLT*LowLtc.yml__ files can be considered the source files for the other configurations.
+For the moment, the __LLT*LowLtc.tmpl__ files can be considered the source files for the other configurations.
 
 To generate the other configurations, run the following commands from the root directory:
 
 ```bash
-for file in src/workloads/transactions/LLT*LowLtc.yml; do
+out_dir=src/workloads/transactions/templates/out
+mkdir -pv $out_dir
+for file in src/workloads/transactions/templates/LLT*LowLtc.tmpl; do
+    cp $file $out_dir/$(basename ${file/tmpl/yml})
+done
+for file in $out_dir/LLT*LowLtc.yml; do
     sed -e 's/^  GlobalRateValue: .*$/  GlobalRateValue: \&GlobalRateValue 1 per 1250 microsecond/' \
```

Comment:
Ah, my eyes! I thought we got rid of all the sed in our codebase. 
I'm not sure I want to ask for a change. Maybe I'll pretend I didn't see this.

Reply:
This is really only for convenience. These files are essentially mothballed and should not be used unless there is a compelling need.